src/test/EC3ConfigurationsClient.test.ts

```diff
@@ -0,0 +1,229 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import * as chaiAsPromised from "chai-as-promised";
+import { expect, use } from "chai";
+import * as sinon from "sinon";
+import { EC3ConfigurationsClient } from "../reporting/clients/EC3ConfigurationsClient";
+import { EC3Configuration, EC3ConfigurationCreate, EC3ConfigurationUpdate } from "../reporting/interfaces/EC3Configurations";
+use(chaiAsPromised);
+
+describe("EC3ConfigurationsClient", () => {
+  const configurationsClient: EC3ConfigurationsClient = new EC3ConfigurationsClient();
+  let fetchStub: sinon.SinonStub;
+  let requestStub: sinon.SinonStub;
+
+  beforeEach(() => {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    fetchStub = sinon.stub(configurationsClient, "fetchJSON" as any);
```

Comment:
Don't stub methods in the `beforeEach` if you are changing them in each test.

Reply:
I am only changing them in the single test that changes the base path of the client (because a new client is created ant therefore needs to be stubbed)